api/app/tests/test_get_hourlies.py

```diff
@@ -36,19 +67,19 @@ def response(monkeypatch, mock_env_with_use_wfwx, mock_jwt_decode, codes):
 
 
 # pylint: disable=redefined-outer-name
-@then('the response status code is <status>')
+@ then('the response status code is <status>')
```

Comment:
I wonder why this happens in your code editor

Reply:
So weird - I didn't want to fight it, and now it's stopped doing it.